packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -178,6 +194,20 @@ export const Alert: React.FunctionComponent<AlertProps> = ({
       onMouseLeave={myOnMouseLeave}
       {...props}
     >
+      {isExpandable && (
+        <div className={css(styles.alertToggle)}>
+          <Button
+            variant={ButtonVariant.plain}
+            onClick={onToggleExpand}
+            aria-expanded={isExpanded}
+            aria-label={`Toggle ${ariaLabel} description`}
```

Comment:
We should not hardcode anything in english. So ideally, we'd expose a separate aria-label for the toggle button and use a well written value as the default value.

Reply:
In core we give the toggle button `aria-label="Details"`, then give an `id` to both the toggle button and alert title, then use `aria-labelledby="[id of title] [id of toggle button]"`

Reply:
I updated the aria label to be customizable and fallback on a default similar to `AlertActionCloseButton`, also put the toggle button in its own component. So the main difference between the two buttons at this point is that the user can not specify their own callback for toggling, like the onClose action in  `AlertActionCloseButton`.